src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Android.cs

```diff
@@ -0,0 +1,122 @@
+﻿#if __ANDROID_29__
+using AndroidX.Core.View;
+#else
+using Android.Support.V4.View;
+#endif
+using System;
+using System.ComponentModel;
+using Android.Content;
+using Android.Graphics.Drawables;
+using Android.Runtime;
+using Java.IO;
+using Android.Graphics;
+using Windows.UI.Xaml.Media;
+using Uno.Extensions;
+using Microsoft.Extensions.Logging;
+using Uno.Logging;
+using Uno.UI.Controls;
+using Uno.UI;
+using Android.Views;
+
+namespace Windows.UI.Xaml.Media
+{
+	/// <summary>
+	/// Renderer to update all frames with better shadows matching material design standards.
+	/// </summary>	
+	public partial class AcrylicBrush
+	{
+		private GradientDrawable _mainDrawable;
+		private GradientDrawable _acrylicLayer;
+
+		protected void UpdateProperties()
+		{
+			UpdateCornerRadius();
+			UpdateAcrylicGlowColor();
+			UpdateAndroidBlurOverlayColor();
+			UpdateAndroidBlurRadius();
+			//UpdateAndroidBlurRootElement();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)